### ISSUE TITLE
[Mobile] Support action based message extensions triggered from ... menu

### ISSUE THREAD

**Issue by matthiasdilger** [MicrosoftDocs/msteams-docs#5102]
This issue has been reported before (#1759), could you please share the current status? It would be really helpful to have this functionality.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7fd5f91c-206f-f03c-d82e-3bf1cc81d13e
* Version Independent ID: d5c2d24a-def4-af2e-7983-c73cf7f22934
* Content: [Messaging extensions - Teams](https://docs.microsoft.com/en-us/microsoftteams/platform/messaging-extensions/what-are-messaging-extensions?tabs=dotnet)
* Content Source: [msteams-platform/messaging-extensions/what-are-messaging-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/main/msteams-platform/messaging-extensions/what-are-messaging-extensions.md)
* Product: **msteams**
* GitHub Login: @surbhigupta
* Microsoft Alias: **anclear**

**Comment by v-chetsh** [comment]
@matthiasdilger - We will check it internally and get back to you.

**Comment by Meghana-MSFT** [comment]
@matthiasdilger - We got an update from the engineering team that Message Actions has been rolled out. It will be available publicly once rollout is complete.

**Comment by matthiasdilger** [comment]
I can see this is now supported now on both Android and iOS, thank you!
There are a lot of issues though, I will open another ticket for that.
